mirrord-agent/src/file.rs

```diff
@@ -17,11 +17,15 @@ use tracing::{debug, error};
 
 use crate::{error::AgentError, runtime::get_container_pid, sniffer::DEFAULT_RUNTIME, PeerID};
 
-// TODO: To help with `openat`, instead of `HashMap<_, File>` this should be
-// `HashMap<_, RemoteFile`>, where `RemoteFile` is a struct that holds both the `File` + `PathBuf`.
+#[derive(Debug)]
+pub struct RemoteFile {
```

Comment:
why do we need it..?

Reply:
when openat is used, we just need a different object for it, but that object doesn't need a path either..